views/default/admin.css.php

```diff
@@ -27,31 +26,33 @@
 
 	.elgg-module-admin-notices {
 		margin-top: 2rem;
-	}
-	
-	.elgg-admin-notices > li {
-		padding: 0;
-		border: none;
+		
+		> li {
+			padding: 0;
+			border: none;
+		}
 	}
 
 	.elgg-page-topbar {
 		background: #2d3047;
+		padding: 0 1rem;
 		
-		 .elgg-menu li .elgg-child-menu {
+		.elgg-menu-container {
+			justify-content: space-between;
+			width: 100%;
+		}
+		
+		.elgg-menu li .elgg-child-menu {
 			background: #2d3047;
 		}
```

Comment:
this could be inside the `.elgg-menu-container` right?

Reply:
fixed